manifests/kubeweekly-jobs.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: kubeweekly-jobs
```

Comment:
for clarity better to add `namespace: default` explicitely

Reply:
addressed here https://github.com/cloudnative-id/kubeops/pull/3/commits/f08dc065397332554b19424289a1ca70a06d156d